rules/gather_licenses_info.bzl

```diff
@@ -110,26 +110,31 @@ def write_licenses_info(ctx, deps, json_out):
       }}"""
 
     licenses = []
+    rules = [] 
```

Comment:
Why not a dict?

Reply:
`rules` is the key and `licenses` is value? nice idea, I will refactor it.